### ISSUE TITLE
Evans not working on Alpine Linux

### ISSUE THREAD

**Issue by kamatama41** [ktr0731/evans#177]
I was trying to run evans on Apline (by Docker), but the binary returns the `"no such file or directory"` error.

**Dockerfile**
```dockerfile
FROM alpine:3.9
RUN apk add --no-cache \
    curl

RUN curl -LO https://github.com/ktr0731/evans/releases/download/0.7.3/evans_linux_amd64.tar.gz \
 && tar zxvf evans_linux_amd64.tar.gz \
 && mv evans /usr/local/bin/ \
 && rm evans_linux_amd64.tar.gz

CMD ["/usr/local/bin/evans", "--help"]
```

**Run result**
```
$ docker build -t evans-run-test . && docker run evans-run-test
...
Successfully tagged evans-run-test:latest
standard_init_linux.go:207: exec user process caused "no such file or directory"
```

On the other hand, It can be run on Ubuntu expectedly

**Dockerfile**
```dockerfile
FROM ubuntu:18.04

RUN apt-get update && apt-get install -y \
  curl

RUN curl -LO https://github.com/ktr0731/evans/releases/download/0.7.3/evans_linux_amd64.tar.gz \
 && tar zxvf evans_linux_amd64.tar.gz \
 && mv evans /usr/local/bin/ \
 && rm evans_linux_amd64.tar.gz

CMD ["/usr/local/bin/evans", "--help"]
```

**Run result**
```
$ docker build -t evans-run-test . && docker run evans-run-test
...
Successfully tagged evans-run-test:latest
evans 0.7.3

Usage: evans [--help] [--version] [options ...] [PROTO [PROTO ...]]

Positional arguments:
        PROTO                   .proto files

Options:
        --edit, -e                     edit config file using by $EDITOR (default "false")                                        
        --repl                         launch Evans as REPL mode (default "false")                                                
        --cli                          start as CLI mode (default "false")                                                        
        --silent, -s                   hide splash (default "false")                                                              
        --host string                  gRPC server host                                                                           
        --port, -p string              gRPC server port (default "50051")                                                         
        --package string               default package                                                                            
        --service string               default service                                                                            
        --call string                  call specified RPC by CLI mode                                                             
        --file, -f string              a script file that will be executed by (used only CLI mode)                                
        --path strings                 proto file paths (default "[]")                                                            
        --header stringToString        default headers that set to each requests (example: foo=bar) (default "[]")                
        --web                          use gRPC Web protocol (default "false")                                                    
        --reflection, -r               use gRPC reflection (default "false")                                                      
        --verbose                      verbose output (default "false")                                                           
        --tls, -t                      use a secure TLS connection (default "false")                                              
        --cacert string                the CA certificate file for verifying the server                                           
        --cert string                  the certificate file for mutual TLS auth. it must be provided with --certkey.              
        --certkey string               the private key file for mutual TLS auth. it must be provided with --cert.                 
        --servername string            override the server name used to verify the hostname (ignored if --tls is disabled)        
        --insecure, -k                 use an insecure connection (ignored if --tls is enabled) (default "true")                  
        --version, -v                  display version and exit (default "false")                                                 
        --help, -h                     display help text and exit (default "false")  
```

**Comment by ktr0731** [comment]
Thank you for reporting an issue.
This issue was fixed in #178 and it is going to tag in the next release.

**Comment by kamatama41** [comment]
Confirmed it works as expected! Thank you!
